.github/workflows/ci.yml

```diff
@@ -21,5 +21,7 @@ jobs:
       - run: npm ci
       - run: npm run check-format
       - run: npm run build
+        env:
+          ALLOWED_LOGIC_URL_PREFIXES: https://prefix.example/
```

Comment:
This would be clearer if you defined more than one

Reply:
It's not really intended as an example, it's just because if I don't put anything here then it won't compile at all. The actual documentation is in the README.

Reply:
Ended up just using the empty string, which suffices when building just to make sure it builds (though the resulting frame would immediately error out if you tried to run it).